### PR TITLE
SSL Listener support

### PR DESCRIPTION
**Pull request by ghost**
It's hard to hate this code, it's mostly metadata.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -4,7 +4,7 @@
 from setuptools import setup, find_packages
 setup(
     name = "a10-neutronclient",
-    version = "0.1.5a",
+    version = "0.1.6a",
```
Comment (dougwig) [inline setup.py]: Don't bump versions as part of functionality changes. We'll just end up double-bumping.

File: a10_neutronclient/a10_certificate.py [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+    shell_command = 'a10-certificatebinding-list'
+
+    list_columns = ['id', 'certificate_id', 'listener_id']
+
+
+class CertificateBindingCreate(client_extension.Create, CertificateBindingExtension):
+    """Create A10 scaling group worker"""
+
+    shell_command = 'a10-certificatebinding-create'
+    list_columns = ['certificate_id', 'listener_id']
```
Comment (Cedev) [inline a10_neutronclient/a10_certificate.py]: Minor: should have `id` column

File: a10_neutronclient/a10_certificate.py [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+    list_columns = ['id', 'name', 'description']
+
+
+class CertificateCreate(client_extension.Create, CertificateExtension):
+    """Create A10 scaling group"""
+
+    shell_command = 'a10-certificate-create'
+
+    list_columns = ['id', 'name', 'description', 'cert_data', 'key_data', 'intermediate_data', 'password']
```
Comment (Cedev) [inline a10_neutronclient/a10_certificate.py]: We let them get the key_data/password by api?

**Comment by Cedev** [comment]
This feature wasn't added to a10-neutron-lbaas. (See https://github.com/a10networks/a10-neutron-lbaas/pull/354). A certificate manager other than barbican can be configured by configuring the neutron lbaas certificate manager type/class.

**Pull request closed without merging**